File: Python 3.8/1020 - Idade em Dias.py
D = int(input())
A = int(D / 365)
D -= A * 365
M = int(D / 30)
D -= M * 30
print('{} ano(s)'.format(A))
print('{} mes(es)'.format(M))
print('{} dia(s)'.format(D))

File: C#/1020 - Idade em Dias.cs
using System; 

class URI {
  static void Main(string[] args) {
    int D = Convert.ToInt32(Console.ReadLine());
    int ANO = D / 365;
    D -= ANO * 365;
    int MES = D / 30;
    D -= MES * 30;
    Console.WriteLine(ANO + " ano(s)"); 
    Console.WriteLine(MES + " mes(es)");
    Console.WriteLine(D + " dia(s)");
  }
}
